open_spiel/contrib/python/export_graph.py

```diff
@@ -38,7 +38,7 @@ def main(_):
   game = pyspiel.load_game(FLAGS.game)
 
   # Information state length
-  info_state_shape = game.information_state_tensor_shape()
+  info_state_shape = game.observation_tensor_shape()
```

Comment:
I changed it based on @lanctot comment during the extravaganza. With `information_state_tensor_shape()`, that python file gives the following error:

```
Traceback (most recent call last):
  File "open_spiel/contrib/python/export_graph.py", line 103, in <module>
    app.run(main)
  File "/home/ubuntu/open_spiel/venv/lib/python3.8/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/home/ubuntu/open_spiel/venv/lib/python3.8/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "open_spiel/contrib/python/export_graph.py", line 41, in main
    info_state_shape = game.information_state_tensor_shape()
RuntimeError: InformationStateTensorShape unimplemented.
```

Reply:
Ok, makes sense. Can you change the var names / comments to be consistent, i.e. observation_tensor_shape (instead of info_state_shape) in all the places that this change occurs?

Reply:
I changed every `information_state_shape` to `observation_shape` (in Pyhton file) and `info_state_vector` to `observation_shape_vector` in CC file.

Reply:
(and modified the comments)

Reply:
@michalsustr Please check the modifications and then mark this subthread as Resolved. Thank you!